content/components/filter-input.mdx

```diff
@@ -0,0 +1,366 @@
+
+<Note>
+  This component is not available in production yet however multiple teams are currently contributing to this.
+</Note>
+
+## Overview
+
+An input that provides suggestions through qualifiers and highlights complex filter syntax.
+
+## Anatomy
+
+<img
+  width="960"
+  alt="The filter input annotated with specific naming for individual items. The input has a 'label:bug state:open assignee:' value and shows multiple suggestions for users."
+  src="https://user-images.githubusercontent.com/980622/204337049-f8dad70b-efdb-4cea-a33b-eba2c050fb84.png"
+/>
+
+## Content
+
+### Icon
+
+A filter input allows free form text searching as well as using qualifiers to narrow down your search.
+The filter input always supports free form text search. This is why you should use the search icon as a leading visual. A filter icon could give the impression that free form text search isn't available.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/980622/204516414-4e6e00ba-38f7-49e5-8a80-9e3bc91ccd97.png"
+      role="presentation"
+      width="456"
+      alt="A input that has a magnifying glass icon on the left side."
+    />
+    <Caption>Use the search icon in a filter input.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/980622/204516427-75b0a18d-9f61-4541-931c-ba0e8175fa07.png"
+      role="presentation"
+      width="456"
+      alt="A input that has a filter icon on the left side which consists out of 3 lines getting incrementally smaller."
+    />
+    <Caption>Don't use a filter icon in a filter input.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Submit
+
+There are two ways this component can trigger submission of the input value.
+
+1. Provide a search button next to the filter input
+2. Auto update results as the value changes
+
+### Clear button
+
+The clear button only appears when there is at least one character entered in the input. When the input is empty the clear button disappears again.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/980622/204872106-ff9e6eb7-dc9a-421a-a82e-4a3df9115795.png"
+      role="presentation"
+      width="456"
+      alt="A input without a value."
+    />
+    <Caption>Hide the clear button when the input is empty.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/980622/204872101-33720780-4f9a-4141-bffc-4d008dde755e.png"
+      role="presentation"
+      width="456"
+      alt="A input without a value showing a clear button in it on the right."
+    />
+    <Caption>Don't show the clear button when the input is empty.</Caption>
+  </Dont>
+</DoDontContainer>
+
+It's important to stick to an icon for clearing the input to avoid any confusion with the value of the input. Especially when the input has a value that is long and gets close to the clear button this could be an problem if it was a link.
```

Comment:
```suggestion
Use an icon instead of a link for clearing the input. This avoids confusion with the value of the input. Especially when the input has a long value that gets close to the clear button a link could be problematic.
```